SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java

```diff
@@ -57,4 +58,16 @@ public void testUser(){
         assertEquals("JoinColumn user: name is not equal", "user_id", joinColumn.name());
         assertEquals("JoinColumn user: nullable is true", false, joinColumn.nullable());
     }
+
+    @Test
+    public void testGroup() {
+        AssertAnnotations.assertField(Message.class, "group",ManyToOne.class, JoinColumn.class);
+
+        ManyToOne m = ReflectTool.getFieldAnnotation(Message.class, "group", ManyToOne.class);
+        assertEquals("ManyToOne: Fetch is not equal", FetchType.LAZY, m.fetch());
+
+        JoinColumn joinColumn = ReflectTool.getFieldAnnotation(Message.class, "group", JoinColumn.class);
+        assertEquals("JoinColumn group: name is not equal", "group_id", joinColumn.name());
+        assertEquals("JoinColumn group: nullable is true", false, joinColumn.nullable());
+    }
```

Comment:
Looks like we need more tests for the codecov plugin to approve.

Reply:
Is there a way to test code coverage on my local copy before I commit?

Reply:
Thats a good question, I know its integrate with Github so I think for it to run the code coverage report you have to commit. I could be mistaken

Reply:
@VinV5 to view the test coverage locally. Follow the steps below.
1. Execute "mvn clean install" to run the tests and generate the coverage reports.
2. Go to this folder SECP\SECP-service\target\site\jacoco on your computer
3. Execute the index.html file
4. You would see the ui for the test coverage there. 👍